version.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "0.1",
```

Comment:
What will the versions end up looking like here?

Reply:
They'll be 0.1.{height} since last increment.

Reply:
Does that mean that the NuGet package versions will also include the commit height?  I'd prefer to avoid that.

Reply:
Up to you -- I've found that no one seems to care about the patch version; you can see it in action in any of my libraries (rx/ix, humanizer, etc). It's a way to get a reproducible build/version number that auto versions. Driven by commit depth has been the most reliable way.

I can remove auto versioning if you have another approach you prefer.

Reply:
I like GitVersion personally.  I suppose you can drive it through GitInfo instead.